### ISSUE TITLE
Add aria-label for input field

### ISSUE THREAD

**Issue by f1sh1918** [digitalfabrik/entitlementcard#2005]
### Describe the Problem

Lighthouse reported that some dropdown fields f.e. "Empfänger des Antrags" does not have accessible name on
https://staging.bayern.ehrenamtskarte.app/beantragen

### Describe Your Preferred Solution

- add a aria-label for the dropdown to "Empfänger des Antrags"
- add a aria-label for the dropdown to "Einsatzgebiet"

### Additional Information

[
](https://dequeuniversity.com/rules/axe/4.10/aria-input-field-name)
